src/Stratis.Bitcoin.Features.Miner/Controllers/StakingController.cs

```diff
@@ -123,10 +123,14 @@ public IActionResult StartStaking([FromBody]StartStakingRequest request)
         /// <summary>
         /// Stop staking.
         /// </summary>
+        /// <param name="corsProtection">This body parameter is here to prevent a CORS call from triggering method execution.</param>
+        /// <remarks>
+        /// <seealso cref="https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#Simple_requests"/>
+        /// </remarks>
         /// <returns>An <see cref="OkResult"/> object that produces a status code 200 HTTP response.</returns>
         [Route("stopstaking")]
         [HttpPost]
-        public IActionResult StopStaking()
+        public IActionResult StopStaking([FromBody] bool corsProtection = true)
```

Comment:
What will happen if we don't pass a body here?

Reply:
it's OK, I've tested with this ajax (both with and without content) and the code is not called
```
        $('#ajax_post').click(function () {
            $.ajax({
                type: "POST",
                crossDomain: true,
                headers: {},
                content: {},
                //content: { "corsProtection" : "true" },
                dataType: "json",
                url: "http://localhost:38221/api/Staking/stopstaking",
                success: function (data) {
                    alert(data);
                },
                error: function (error) {
                    alert(error);
                }
            });
        });
```